hack/externalnode/sftp-deployment.yml

```diff
@@ -0,0 +1,34 @@
+apiVersion: apps/v1
```

Comment:
Add a referrence to this file in the SupportBundleCollection introduction document to guide users to deploy SFTP Pod server?

Reply:
updated.